src/main/java/org/folio/circulationbff/service/impl/CirculationBffServiceImpl.java

```diff
@@ -25,6 +26,30 @@ public class CirculationBffServiceImpl implements CirculationBffService {
   private final SettingsService settingsService;
   private final UserTenantsService userTenantsService;
 
+  @Override
+  public StaffSlipCollection fetchPickSlipsByServicePointId(String servicePointId) {
+    log.info("fetchPickSlipsByServicePointId:: servicePointId: {}", servicePointId);
+    boolean isEcsTlrFeatureEnabled = ecsTlrClient.getTlrSettings().getEcsTlrFeatureEnabled();
```

Comment:
Using ecsTlrClient is not enough, mod-circulation-bff will be deployed not only in the Central tenant but also in Data tenants. Please, use SettingsService.